android/src/main/java/com/thebluealliance/androidclient/binders/TeamInfoBinder.java

```diff
@@ -118,6 +125,13 @@ public void updateData(@Nullable TeamInfoBinder.Model data) {
             string.setSpan(new TextAppearanceSpan(mActivity,
                     R.style.InfoItemLabelStyle), 0, 3, Spannable.SPAN_INCLUSIVE_INCLUSIVE);
             teamFullName.setText(string);
+            teamFullName.setMaxLines(TEAM_FULL_NAME_COLLAPSED_MAX_LINES);
+            teamFullName.setEllipsize(TextUtils.TruncateAt.END);
```

Comment:
Perhaps the ideal would ellipsize out the middle to leave the `& school name` tail:

> _aka_ NASA Ames Research Center /
Google / AMEX Plating / Innovation
First ... & Bellarmine College Preparatory

What do you think about using `TextUtils.TruncateAt.START` as an easy approximation?